src/components/Table/Table.stories.mdx

```diff
@@ -55,37 +55,56 @@ NOTE: all properties in a column object are optional, but we highly recommend in
         { heading: 'Description', dataKey: 'description' },
       ];
       return (
-        <Table 
-          rowKey="name" 
-          columns={columnConfig} 
-          rows={columnInterfaceRows} 
+        <Table
+          rowKey="name"
+          columns={columnConfig}
+          rows={columnInterfaceRows}
         />
       );
     }}
   </Story>
 </Canvas>
 
-## Default
-Basic example showcasing the minimum props required to render a table. `rows` `columns` and a `rowKey`.
+## Common Example
+An example showcasing the props required to render a list of customers in a table. `rows` `columns` and a `rowKey`.
 
 <Canvas>
   <Story name="Default">
     {() => {
       const columnConfig = [
-        { heading: 'ID', dataKey: 'id' },
-        { heading: 'Color', dataKey: 'color' },
-        { heading: 'Flavor', dataKey: 'flavor' },
+        { heading: 'Customer', dataKey: 'customerInfo', render: (cell, row, index) => (
+          <div>
+            <div>{cell.name}</div>
+            <div className="font-color-grey">{cell.email}</div>
+          </div>
+         ) },
+        { heading: 'phone', dataKey: 'phone', width: 125 },
+        { heading: 'plan', dataKey: 'plan', width: 125 },
+        { heading: 'status', dataKey: 'status', width: 125 },
       ];
       const tableData = [
-        { id: 1, color: 'red', flavor: 'vanilla' },
-        { id: 2, color: 'blue', flavor: 'chocolate' },
-        { id: 3, color: 'green', flavor: 'strawberry' },
+        { status: 'lead captured',
+          customerInfo: { name: 'John Smith', email: 'john@palmetto.com' },
```

Comment:
@juanfabrega @JCarmine in order to combine properties of a "customer" object like name and email address into a single cell, do we have to combine them into an object first (in this example i'm combining them into a `customerInfo` object). 

I want to able to render tables like this

<img width="719" alt="Screen Shot 2020-09-01 at 4 42 17 PM" src="https://user-images.githubusercontent.com/1447339/91916488-1cb0d800-ec72-11ea-83b4-28b135f759c5.png">

Reply:
@nathanyoung you do not have to combine them in your original data structure. Instead you can use the `render` method in the column config, which has access to the whole row. The render method will take 3 arguments `(cell, row, rowIndex)`. Here is an example of a column config: 

```
renderCustomer = (cell, row) => (
  <div>
    <p>{row.name}</p>
    <p className="text-grey-light">{row.email}</p>
  </div>
);

columns: [
 { heading: 'CUSTOMER', render: renderCustomer },
  ...restOfColumns
];
```

You can use the column render method to:
* Combine fields (forgoing the `dataKey` altogether.
* Shape data. E.G: transform primitive number values into currency strings.
* Style data. E.G: Create a button or link based on the cell data or row, or index.
* Render something completely unrelated to the data at hand, if you need to.